turbo.json

```diff
@@ -6,13 +6,16 @@
       "inputs": ["$TURBO_DEFAULT$", ".env*"],
       "outputs": ["dist/**"]
     },
-    "//#check": {},
-    "//#format": {
+    "//#lint": {},
+    "//#lint:fix": {
       "cache": false
     },
     "dev": {
       "cache": false,
       "persistent": true
+    },
```

Comment:
Dev command looks good, but you've not got a build command for when you just want to build everything and not run it as dev.

You'll need build and maybe publish commands later if/when you decide to publish to NPM or something.